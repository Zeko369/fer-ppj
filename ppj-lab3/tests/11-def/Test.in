<program>
 <lista_naredbi>
  <naredba>
   <za_petlja>
    KR_ZA 1 za
    IDN 1 i
    KR_OD 1 od
    <E>
     <T>
      <P>
       BROJ 1 1
      <T_lista>
       $
     <E_lista>
      $
    KR_DO 1 do
    <E>
     <T>
      <P>
       IDN 1 i
      <T_lista>
       $
     <E_lista>
      $
    <lista_naredbi>
     <naredba>
      <naredba_pridruzivanja>
       IDN 2 x
       OP_PRIDRUZI 2 =
       <E>
        <T>
         <P>
          BROJ 2 3
         <T_lista>
          $
        <E_lista>
         $
     <lista_naredbi>
      $
    KR_AZ 3 az
  <lista_naredbi>
   $
