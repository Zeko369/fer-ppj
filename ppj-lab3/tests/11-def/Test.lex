KR_ZA 1 za
IDN 1 i
KR_OD 1 od
BROJ 1 1
KR_DO 1 do
IDN 1 i
IDN 2 x
OP_PRIDRUZI 2 =
BROJ 2 3
KR_AZ 3 az
