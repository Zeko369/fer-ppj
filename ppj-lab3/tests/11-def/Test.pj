za i od 1 do i
 x = 3
az
