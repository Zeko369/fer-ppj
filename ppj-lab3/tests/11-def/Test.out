err 1 i
